src/main.py

```diff
@@ -240,7 +252,16 @@ def get_dataset_edition_by_id(
         request_id=request_id,
     )
 
-    if request.headers["Accept"] == JSONLD or BROWSABLE:
+    if request.headers["Accept"] == CSV:
```

Comment:
Can you add a comment above this to explain what it is. eg "If someone is asking for a csv at the edition level we return the latest version of the this edition.

If we come back and read this at a future date that'll be a really handy thing to know.